src/celengine/starname.cpp

```diff
@@ -18,6 +18,10 @@ using namespace std;
 
 uint32_t StarNameDatabase::findCatalogNumberByName(const string& name) const
 {
+    uint32_t catalogNumber = getCatalogNumberByName(name);
+    if (catalogNumber != Star::InvalidCatalogNumber)
+        return catalogNumber;
+
     string priName   = name;
```

Comment:
As you removed some extra spaces in not changed lines, then please remove it here as well.